src/brain/issueLabelHandler/project.ts

```diff
@@ -0,0 +1,48 @@
+import { EmitterWebhookEvent } from '@octokit/webhooks';
+import * as Sentry from '@sentry/node';
+import { ISSUES_PROJECT_NODE_ID, PRODUCT_AREA_FIELD_ID } from '@/config';
+import { getIssueDetailsFromNodeId, getProductAreaFromProjectField } from './helpers';
+import { getClient } from '@api/github/getClient';
+import { ClientType } from '@/api/github/clientType';
+import { PRODUCT_AREA_LABEL_PREFIX } from '@/config';
+
+function checkForIssuesProject(payload) {
+    return payload?.projects_v2_item?.project_node_id === ISSUES_PROJECT_NODE_ID;
+}
+
+function checkForProductAreaField(payload) {
+    return payload?.changes?.field_value?.field_node_id === PRODUCT_AREA_FIELD_ID;
+}
+
+function checkForProjectItemNodeId(payload) {
+    return payload?.projects_v2_item?.node_id == null || payload?.projects_v2_item?.content_node_id == null;
+}
+
+export async function syncLabelsWithProjectField({
+    id,
+    payload,
+    ...rest
+}: EmitterWebhookEvent<'projects_v2_item.edited'>) {
+    const tx = Sentry.startTransaction({
+        op: 'brain',
+        name: 'issueLabelHandler.syncLabelsWithProjectField',
+    });
+
+    if (!checkForIssuesProject(payload) || !checkForProductAreaField(payload)|| checkForProjectItemNodeId(payload)) {
```

Comment:
u no like `shouldSkip` 😏

Reply:
oh woops, I initially did it this way since I think I was refactoring some logic, I can add this back so it stays consistent